File: build-system/integration-test/test-projects/sourceDependency/app/src/main/java/com/example/android/multiproject/MainActivity.java
package com.example.android.multiproject;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import com.example.android.multiproject.library.ShowPeopleActivity;

public class MainActivity extends Activity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
    }

    public void sendMessage(View view) {
        Intent intent = new Intent(this, ShowPeopleActivity.class);
        startActivity(intent);
    }
}

File: build-system/integration-test/test-projects/databindingAndKotlin/app/src/main/kotlin/com/example/android/kotlin/MainActivity.kt
package com.example.android.kotlin

import android.app.Activity
import android.databinding.DataBindingUtil
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import com.example.android.kotlin.databinding.ActivityLayoutBinding

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding : ActivityLayoutBinding = DataBindingUtil.setContentView(this, R.layout.activity_layout)
        binding.model = ViewModel("foo", "bar")
    }
}
